packages/core/src/rum/instrumentation/DdRumUserInteractionTracking.tsx

```diff
@@ -65,7 +65,7 @@ export class DdRumUserInteractionTracking {
         ) => {
             return originalMemo(component, (prev, next) => {
                 if (!next.onPress || !prev.onPress) {
-                    return !!propsAreEqual
```

Comment:
What is the motivation behind this change? Why we used `!!` (well, probably it is a conversion from `null|undefined` to boolean) and now it is not needed?

Reply:
Because we chain this to a ternary operator (`?`), so using `!!` actually does not make any difference here and can be removed. I believe I should not have added it in the first place.